tests/TestCase/View/Helper/FormHelperTest.php

```diff
@@ -2433,8 +2437,13 @@ public function testErrorMessageDisplay(): void
             'Title',
             '/label',
             'input' => [
-                'type' => 'text', 'name' => 'Article[title]',
-                'id' => 'article-title', 'class' => 'form-error',
+                'type' => 'text',
+                'name' => 'Article[title]',
+                'id' => 'article-title',
+                'class' => 'form-error',
+                'aria-invalid' => 'true',
+                // TODO this seems wrong as the id doesn't exist.
+                'aria-describedby' => 'article-title-error',
```

Comment:
There are a few tests I marked with TODO as the results look off to me.

Reply:
The `control()` call should probably disable the attribute, ie pass `'aria-describedby' => null`.